starter-code/routes/index.js

```diff
@@ -1,9 +1,76 @@
-const express = require('express');
-const router  = express.Router();
+const express = require("express");
+const router = express.Router();
+const userModel = require("./../models/user");
+const bcrypt = require("bcrypt");
+// const createUser = require("./../views/create-user");
 
 /* GET home page */
-router.get('/', (req, res, next) => {
-  res.render('index');
+router.get("/", (req, res, next) => {
+  res.render("index");
+});
+
+router.get("/user", (req, res) => {
+  res.render("user");
+});
+
+//signin
+
+router.post("/create-user", (req, res, next) => {
```

Comment:
if think you mistaken between signup(create a new user) and signin/login(connect to your account which is laready created)